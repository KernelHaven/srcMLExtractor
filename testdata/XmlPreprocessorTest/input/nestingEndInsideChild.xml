<unit language="C">
	<cpp:ifdef>#<cpp:directive>ifdef</cpp:directive><name>A</name></cpp:ifdef>
	<block>
		<empty_stmt>S1</empty_stmt>
		<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
		<empty_stmt>S2</empty_stmt>
	</block>
</unit>
