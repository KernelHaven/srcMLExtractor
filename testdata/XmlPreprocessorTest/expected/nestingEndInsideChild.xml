<unit lineStart="0" lineEnd="7" language="C">
	<cpp:ifdef lineStart="1" lineEnd="4">#<cpp:directive>ifdef</cpp:directive><name>A</name>
		<block>
			<empty_stmt>S1</empty_stmt>
			
			<kh:reference></kh:reference>
		</block>
	</cpp:ifdef>
	<empty_stmt lineStart="5" lineEnd="5">S2</empty_stmt>
</unit>
